### DISCUSSION TITLE
Scaling the chromedriver

### DISCUSSION THREAD

**Discussion by watchingdogs** [ultrafunkamsterdam/undetected-chromedriver#1138] [Q&A]
What is the intended way the to scale the chromedriver? Spawning new threads? Making multiple browser windows? I have seen multiple solutions to this problem, but I am not sure which one is the official and which is the hacked-together-kinda-works solution.

**Comment by bbkinglisbon** [comment]
I'm also looking for this

**Reply by jcgohr** [reply]
Same, I think multiple threaded objects would be the way to go

**Comment by dempc6469** [comment]
Personally, and I should preface that I'm novice at best in Python, I run `concurrent.futures.ProcessPoolExecutor ` . I have 4 sites that I scrape continuously with random wait times in between runs. Each site has 20-30 URLs that I load in separate tabs. I let chrome do it's thing with threading/tabs and just focus on keeping each site scrape loop tied to a proc. 

YMMV ... and this may be the totally wrong approach but it seems to work for my needs. I would be interested to hear others approach on this.
